### ISSUE TITLE
Can't contact LDAP server for binding ... check you binding

### ISSUE THREAD

**Issue by sock3t** [Kipjr/ldap_login#12]
While using the Ldap_Login Test for a regular user it fails with
`Error : Can't contact LDAP server for binding on server ldaps://dc.ad.domain.com:636 for user piwigo, check your binding!`
This is from /var/log/Ldap_login.log:
```
2018-09-19T14:24:16+02:00: [function]> ldap_search_dn(piwigo)
2018-09-19T14:24:16+02:00: [ldap_search_dn]> Connecting to server
2018-09-19T14:24:16+02:00: [ldap_search_dn]> make_ldap_bind_as($this->cnx,CN=piwigo ldap user,OU=system,OU=accounts,OU=items,DC=ad,DC=domain,DC=com,*mysecretpwhere*)
2018-09-19T14:24:16+02:00: [function]> make_ldap_bind_as
2018-09-19T14:24:16+02:00: [make_ldap_bind_as]> $conn,CN=piwigo ldap user,OU=system,OU=accounts,OU=items,DC=ad,DC=domain,DC=com,*mysecretpwhere*
2018-09-19T14:24:16+02:00: [ldap_search_dn]> Cannot bind to server!
2018-09-19T14:24:16+02:00: [function]> ldap_bind_as
2018-09-19T14:24:16+02:00: [ldap_bind_as]> CN=piwigo ldap user,OU=system,OU=accounts,OU=items,DC=ad,DC=domain,DC=com,*mysecretpwhere*
2018-09-19T14:24:16+02:00: [function]> make_ldap_bind_as
2018-09-19T14:24:16+02:00: [make_ldap_bind_as]> $conn,CN=piwigo ldap user,OU=system,OU=accounts,OU=items,DC=ad,DC=domain,DC=com,*mysecretpwhere*
```
I am running an Active Directory Domain Controller based on SAMBA 4.
I am using SSL protected LDAP. For that I have copied the ca-certificate of that DC into `/usr/local/share/ca-certificates/` of the piwigo server because that was required for another webapp that successfully authenticates against the same DC (nextcloud).

Any network related issue like DNS or Firewall can be ruled out because I can see the ldap traffic on the DC side through tcpdump.

Can we make the _debug_ log file more verbose?

**Comment by sock3t** [comment]
Btw. just checked to bind with plain ldap (no ssl) - same issue.
Also retried with the ca-certificated now listed in /etc/ssl/certs/ca-certificates.crt which is referenced in /etc/ldap/ldap.conf - still same error messages

**Comment by sock3t** [comment]
Hi @Kipjr,

this has been solved for me. The bind works now.

Thanks a lot!

**Comment by Kipjr** [comment]
How did you solve it? 

> 
> 
> Hi @Kipjr,
> 
> this has been solved for me. The bind works now.
> 
> Thanks a lot!

May I ask how it was solved?

**Comment by sock3t** [comment]
I did not change anything on my setup as far as I can tell. I was able to reproduce the same error message by providing a wrong `LDAP server host`. 

But as I said back in Sep.: I checked DNS & IP connectivity etc. could even prove (tcpdump) that my piwigo was communicating with that SAMBA DC. At the same time other applications authenticated just fine against it.

So it would be good to know how to increase debug level, also for the other issue I have.

Thanks
